example_app_generator/spec/support/default_preview_path

```diff
@@ -37,12 +37,18 @@ require_file_stub 'config/environment' do
     module ExampleApp
       class Application < Rails::Application
         config.eager_load = false
+        if Rails::VERSION::STRING.start_with?('7')
+          config.active_support.cache_format_version = 7.0
+        end
 
         # Don't care if the mailer can't send.
         config.action_mailer.raise_delivery_errors = false unless ENV['NO_ACTION_MAILER']
-
         if ENV['CUSTOM_PREVIEW_PATH']
-          config.action_mailer.preview_path = ENV['CUSTOM_PREVIEW_PATH']
+          if Rails::VERSION::STRING.start_with?('7.1')
```

Comment:
@JonRowe Won't this fail with Rails 7.2+? Would it be better to do a `>=` comparison?